.travis.yml

```diff
@@ -11,12 +11,10 @@ cache:
   pip: true
   yarn: true
   directories:
-    - frontend/node_modules
 before_install:
   - sudo apt-key adv --fetch-keys http://dl.yarnpkg.com/debian/pubkey.gpg
   - echo "deb http://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
   - sudo apt-get update -qq
-  - sudo apt-get install -y -qq yarn
```

Comment:
if we no longer need to install yarn then this whole before_install section is not needed

Reply:
> if we no longer need to install yarn then this whole before_install section is not needed

done :)